tests/js/spec/components/collapsible.spec.jsx

```diff
@@ -1,4 +1,4 @@
-import {mountWithTheme} from 'sentry-test/enzyme';
+import {fireEvent, mountWithTheme, screen} from 'sentry-test/reactTestingLibrary';
```

Comment:
since the PR https://github.com/getsentry/sentry/pull/29638 was just merged... can we please replace `fireEvent` with `userEvent`? 🙏